src/routes/builds.ts

```diff
@@ -7,16 +7,17 @@ const router = express.Router();
 
 // Given a buildId corresponding to a persistence module build/upload, return all
 // documents (page ASTs, metadata, assets) associated for that build
-router.get('/:buildId/documents', async (req, res, next) => {
+export const docsPerBuildIdHandler: RequestHandler = async (req, res, next) => {
```

Comment:
(Nit; non-blocking) Noticed that we removed the separate request handler functions for the routes in projects. Should we do it here as well?

Reply:
good catch :+1: removed